lib/services/getRunDefinition.js

```diff
@@ -52,27 +51,28 @@ const getRunDefinition = (run) => {
 
         // Physics or cosmics
         if ((detectors && detectors?.includes('ITS') || detectors?.includes('FT0')) && stableBeamsStart) {
-            definition = RunDefinition.Physics;
-        } else {
-            definition = RunDefinition.Cosmic;
-        }
-    } else {
-        // Technical or Synthetic
-        if (runType?.name?.toUpperCase() === 'TECHNICAL' && pdpBeamType === 'technical') {
-            definition = RunDefinition.Technical;
-        } else if (
-            !dcs
-            && triggerValue === 'OFF'
-            && readoutCfgUri?.match(/^file:\/\/\/.*replay.*\/cfg$/)
-            && readoutCfgUri?.match(/^file:\/\/\/.*(pp|pbpb).*\/cfg$/)
-            && !pdpWorkflowParameters?.includes('CTF')
-        ) {
-            definition = RunDefinition.Synthetic;
-        } else if (runType?.name?.toUpperCase().startsWith('CALIBRATION_')) {
-            definition = RunDefinition.Calibration;
+            return RunDefinition.Physics;
+        } else if (runType?.name.match(/^cosmics?$/i)) {
+            return RunDefinition.Cosmic;
         }
     }
 
+    let definition = null;
+
+    if (runType?.name?.toUpperCase() === 'TECHNICAL' && pdpBeamType === 'technical') {
```

Comment:
Yes, because before if the run matched the previous part of the criteria it was either physics or either cosmics, for sure. But now, it can match the common criteria but none of the specific ones, and so we need to continue